src/components/HeaderKebab.jsx

```diff
@@ -91,10 +93,47 @@ const AnacondaAboutModal = ({ isModalOpen, setIsAboutModalOpen }) => {
     );
 };
 
+const CockpitNetworkConfiguration = ({ setIsNetworkOpen }) => {
+    const idPrefix = "cockpit-network-configuration";
+
+    const handleClose = () => {
+        setIsNetworkOpen(false);
+    };
+
+    return (
+        <Modal
+          aria-label={_("Configure network")}
+          isOpen
+          onClose={handleClose}
+          showClose={false}
+          className={idPrefix + "-modal-page-section"}
+          variant={ModalVariant.large}>
+            <ModalHeader title={_("Network Configuration")} />
+            <ModalBody>
+                <div className={idPrefix + "-page-section-cockpit-network"}>
+                    <PageSection hasBodyWrapper={false}>
+                        <iframe
+                          src="/cockpit/@localhost/network/index.html"
+                          name="cockpit-network"
+                          id="cockpit-network-frame"
+                          className={idPrefix + "-iframe-cockpit-network"} />
+                    </PageSection>
+                </div>
+            </ModalBody>
+            <ModalFooter>
+                <Button variant="secondary" onClick={handleClose}>
+                    {_("Close")}
+                </Button>
+            </ModalFooter>
+        </Modal>
+    );
+};
+
 export const HeaderKebab = ({ currentStepId, dispatch, isConnected, onCritFail, reportLinkURL, setShowStorage, showStorage }) => {
     const [isOpen, setIsOpen] = useState(false);
     const [isAboutModalOpen, setIsAboutModalOpen] = useState(false);
     const [isReportIssueOpen, setIsReportIssueOpen] = useState(false);
+    const [isNetworkOpen, setIsNetworkOpen] = useState(false);
```

Comment:
I think you need to mimic how we handle the cockpit storage dialog open, (showStorage)
We need to know that the network dialog is open from the Error components, so that we mark crashes in the cockpit-networkmanager as such in the bug reports.